api/naming/naming.go

```diff
@@ -9,6 +9,11 @@ import (
 	"github.com/docker/swarmkit/api"
 )
 
+const (
+	// PredefinedLabel identifies internally allocated swarm resources
+	PredefinedLabel = "com.docker.swarm.predefined"
```

Comment:
I can keep the label in `networkallocator` pkg, but then, the docker/cluster code will need to import that package when looking for the label, when the label itself is not really specific to network object only. In future it could be used for any other swarm resource that swarm may self create ?